### ISSUE TITLE
db exec error

### ISSUE THREAD

**Issue by KnoAll** [Cacti/plugin_thold#246]
Seeing recurring errors for thold. I've just updated via git....
This appears to be limited to just one of my cacti installs at the moment

```
2019/02/12 10:02:39 - CMDPHP ERROR: A DB Exec Failed!, Error: Table 'cacti.thold_data' doesn't exist in engine
--
2019/02/12 10:02:39 - CMDPHP PHP ERROR WARNING Backtrace: (/poller.php[796]:api_plugin_hook(), /lib/plugins.php[74]:api_plugin_run_plugin_hook(), /lib/plugins.php[177]:thold_poller_bottom(), /plugins/thold/includes/polling.php[45]:thold_check_all_thresholds(), /plugins/thold/includes/polling.php[464]:CactiErrorHandler())
2019/02/12 10:02:39 - ERROR PHP WARNING in Plugin 'thold': Invalid argument supplied for foreach() in file: /var/www/html/cacti/plugins/thold/includes/polling.php on line: 464
```

**Comment by netniV** [comment]
There was an update to Thresholds yesterday which corrected an installation issue.  Update again and see if this is already resolved by #245

**Comment by KnoAll** [comment]
after update I disabled uninstalled/reinstalled and it appears to be working without error now. thanks
